<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipPlacementWindow</class>
 <widget class="QDialog" name="ShipPlacementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расстановка кораблей</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                    stop:0 #0a2463, stop:1 #3e92cc);
}

QLabel {
    color: #f8f8ff;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
}

QLabel#labelTitle {
    font-size: 28px;
    font-weight: bold;
    color: #f8f8ff;
    background-color: rgba(30, 61, 107, 0.7);
    border-radius: 10px;
    padding: 5px 10px;
    border: 2px solid #d4af37;
}

QLabel#labelShip1Count,
QLabel#labelShip2Count,
QLabel#labelShip3Count,
QLabel#labelShip4Count {
    font-size: 14px;
    color: #f8f8ff;
    font-weight: bold;
}

QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #1e3d6b, stop:1 #0a2463);
    border: 2px solid #d4af37;
    border-radius: 8px;
    padding: 8px 16px;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    font-size: 14px;
    color: #f8f8ff;
    min-width: 100px;
    min-height: 30px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #3a5f8c, stop:1 #1e3d6b);
}

QPushButton:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #0a2463, stop:1 #1e3d6b);
}

QGroupBox {
    background-color: rgba(30, 61, 107, 0.7);
    border: 2px solid #d4af37;
    border-radius: 10px;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    font-size: 14px;
    color: #f8f8ff;
    padding-top: 15px;
    margin-top: 10px;
}

QPushButton#btnStartBattle {
    min-width: 120px;
    min-height: 40px;
    font-size: 16px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #4CAF50, stop:1 #2E7D32);
    border: 2px solid #81C784;
}

QPushButton#btnStartBattle:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #66BB6A, stop:1 #388E3C);
}

QPushButton#btnStartBattle:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #2E7D32, stop:1 #1B5E20);
}

/* Стили для игровых клеток */
QPushButton[isCell=&quot;true&quot;] {
    background-color: #a3c1da;
    border: 1px solid #1e3d6b;
    border-radius: 4px;
    margin: 1px;
    min-width: 30px;
    min-height: 30px;
    max-width: 30px;
    max-height: 30px;
}

QPushButton[isCell=&quot;true&quot;][occupied=&quot;true&quot;] {
    background-color: #2c3e50;
    border-radius: 4px;
}

QPushButton[isCell=&quot;true&quot;]:hover {
    background-color: #b8d4e7;
}

QPushButton[isCell=&quot;true&quot;]:pressed {
    background-color: #8aa8c1;
}
   </string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Расстановка кораблей</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Корабли</string>
   </property>
   <widget class="QPushButton" name="btnShip1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>136</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>1-палубный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShip2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>136</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2-палубный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShip3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>136</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3-палубный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShip4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>136</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>4-палубный</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelShip1Count">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1-палубные: 0/4</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShip2Count">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2-палубные: 0/3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShip3Count">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3-палубные: 0/2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShip4Count">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4-палубные: 0/1</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>550</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutField">
    <property name="spacing">
     <number>4</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnStartBattle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать бой</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBackToMainMenu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
